app/models/SessionsRepository.scala

```diff
@@ -33,23 +33,37 @@ case class SessionInfo(userId: String,
                        cancelled: Boolean,
                        active: Boolean,
                        expirationDate: BSONDateTime,
+                       youtubeLink: Option[String],
+                       slideShareLink: Option[String],
                        reminder: Boolean = false,
                        notification: Boolean = false,
-                       _id: BSONObjectID = BSONObjectID.generate)
+                       _id: BSONObjectID = BSONObjectID.generate
+                      )
 
 case class UpdateSessionInfo(sessionUpdateFormData: UpdateSessionInformation,
                              expirationDate: BSONDateTime)
 
+case class SessionLinks(email: String,
+                        topic: String,
+                        youtubeLink: Option[String],
+                        slideShareLink: Option[String],
+                        _id: BSONObjectID)
+
 object SessionJsonFormats {
 
   import play.api.libs.json.Json
 
   implicit val sessionFormat = Json.format[SessionInfo]
+  implicit val sessionLinksFormat = Json.format[SessionLinks]
 
   sealed trait SessionState
+
```

Comment:
Please remove extra lines added in between
`sealed trait SessionState`
and 
`case object SchedulingNextUnNotified extends SessionState`